src/functionals.jl

```diff
@@ -1,34 +1,39 @@
-function getEndpoints(distr::UnivariateDistribution, epsilon::Real)
-    (left,right) = map(x -> quantile(distr,x), (0,1))
-    leftEnd = left!=-Inf ? left : quantile(distr, epsilon)
-    rightEnd = right!=-Inf ? right : quantile(distr, 1-epsilon)
-    (leftEnd, rightEnd)
-end
-
-function expectation(distr::ContinuousUnivariateDistribution, g::Function, epsilon::Real)
-    f = x->pdf(distr,x)
-    (leftEnd, rightEnd) = getEndpoints(distr, epsilon)
-    quadgk(x -> f(x)*g(x), leftEnd, rightEnd)[1]
+function expectation(distr::ContinuousUnivariateDistribution, g::Function; kwargs...)
+    return first(quadgk(x -> pdf(distr, x) * g(x), extrema(distr)...; kwargs...))
 end
 
 ## Assuming that discrete distributions only take integer values.
 function expectation(distr::DiscreteUnivariateDistribution, g::Function, epsilon::Real)
```

Comment:
Can you (for consistency) also deprecate `expectation(distr::DiscreteUnivariateDistribution, g::Function, epsilon::Real)` and use
```suggestion
function expectation(distr::DiscreteUnivariateDistribution, g::Function; epsilon::Real=1e-10)
```
instead?